runatlantis.io/docs/api-endpoints.md

```diff
@@ -0,0 +1,151 @@
+# API Endpoints
+
+Aside from interacting via pull request comments, Atlantis could respond to a limited number of API endpoints.
+The API endpoints are disabled by default, since the API endpoint could change the infrastructure directly.
+To enable API endpoints, `api-secret` should be configured.
+
+:::tip Prerequisites
+
+* You have set `api-secret` as part of the [Server Configuration](server-configuration.html#api-secret)
```

Comment:
Let's avoid using the word `you`  if possible in this doc 

Example

```suggestion
* set `api-secret` as part of the [Server Configuration](server-configuration.html#api-secret)
```

Reply:
Got it!